### PR TITLE
[Excel] (Custom functions) Link fix

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3844821](https://github.com/OfficeDev/office-js-docs-pr/pull/1101/commits/3844821f81aef609807fdcbaa8ebde94ba58b544)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906181741231626-1101%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Os9hyr%2Fw89ZlEyjcchwmgSQBx6C8nk2ay8oYhvhLrwE%3D&st=2019-06-18T17%3A37%3A27Z&se=2019-07-19T17%3A42%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e60940e](https://github.com/OfficeDev/office-js-docs-pr/pull/1101/commits/e60940ec9f5ea0cc8516915407dd2fa03e0c78b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906181750151604-1101%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=PhCVGzAolG4UkZ6I5xFZ%2B1FAGoP09a8RaXSjGYA5L6s%3D&st=2019-06-18T17%3A46%3A16Z&se=2019-07-19T17%3A51%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e60940e](https://github.com/OfficeDev/office-js-docs-pr/pull/1101/commits/e60940ec9f5ea0cc8516915407dd2fa03e0c78b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906181750253651-1101%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=P5g4uQ1y8mhcr5sjs%2B%2BViHmnAhwvWEO04S2BTgnh%2B8Q%3D&st=2019-06-18T17%3A47%3A29Z&se=2019-07-19T17%3A52%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/excel-custom-functions-note.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 > [!IMPORTANT]
-> Note that Excel custom functions are available in Office on Windows (version 1904 or later, connected to Office 365 subscription), Office on Mac (version 16.24 or later, connected to Office 365 subscription), and Office on the web. Excel custom functions are not supported in Office 2019 (one-time purchase).
+> The custom functions feature is being released to production! Discover the latest updates to custom functions [in this blog post](https://developer.microsoft.com/en-us/office/blogs/office-extensibility-build-2019/). Note that Excel custom functions are available in Office on Windows (version 1904 or later, connected to Office 365 subscription), Office on Mac (version 16.24 or later, connected to Office 365 subscription), and Office on the web. Excel custom functions are not supported in Office 2019 (one-time purchase).
```
Comment (AlexJerabek) [inline docs/includes/excel-custom-functions-note.md]: Aren't CFs in production? If so, change the first sentence to match that reality (is in production).

Comment (AlexJerabek) [inline docs/includes/excel-custom-functions-note.md]: Please remove the "en-us" from the URL.

Comment (mscharlock) [reply docs/includes/excel-custom-functions-note.md]: So apparently not

Comment (mscharlock) [reply docs/includes/excel-custom-functions-note.md]: Oh right, thanks!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[aa331e5](https://github.com/OfficeDev/office-js-docs-pr/pull/1101/commits/aa331e5a7d496d4ea249696d7d4c6cde184c53e8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-batching.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-batching.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-localize.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-localize.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-parameter-options.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-parameter-options.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-save-state.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-save-state.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-volatile.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/excel/custom-functions-volatile.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-custom-functions-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixmanifest/docs/includes/excel-custom-functions-note.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906201831030483-1101%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=w77zZz%2BsVmwKVWqZ%2FLxd3%2B%2Bso6rYMtZhbiZIIlkc1Eg%3D&st=2019-06-20T18%3A27%3A10Z&se=2019-07-21T18%3A32%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**